react-select/react-select.d.ts

```diff
@@ -5,7 +5,7 @@
 
 /// <reference path="../react/react.d.ts"/>
 
-declare namespace ReactSelect {
+declare namespace ReactSelectNameSpace {
```

Comment:
:-1: _but why!_ this is a significant API break for all consumers (granted just a string rename but still) for which I see no benefit. 

do other packages include the word "NameSpace" in their namespace names? I doubt it.